ui/react-components/components/ListView/ListView.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState, useRef, useEffect } from 'react'
+import classNames from 'classnames';
+import './ListView.module.scss'
+import * as moment from 'moment'
```

Comment:
You should be able to specific import of moment by just mentioning. "import  moment from 'moment'"